osu.Game.Tests/Rulesets/Scoring/ScoreProcessorTest.cs

```diff
@@ -53,5 +53,257 @@ public void TestSingleOsuHit(ScoringMode scoringMode, HitResult hitResult, int e
+        [TestCase(ScoringMode.Standardised, HitResult.SmallTickHit, HitResult.SmallTickHit, 925_000)]
+        [TestCase(ScoringMode.Standardised, HitResult.LargeTickMiss, HitResult.LargeTickHit, 0)]
+        [TestCase(ScoringMode.Standardised, HitResult.LargeTickHit, HitResult.LargeTickHit, 575_000)]
+        [TestCase(ScoringMode.Standardised, HitResult.SmallBonus, HitResult.SmallBonus, 700_030)]
+        [TestCase(ScoringMode.Standardised, HitResult.LargeBonus, HitResult.LargeBonus, 700_150)]
+        [TestCase(ScoringMode.Classic, HitResult.Miss, HitResult.Great, 0)]
+        [TestCase(ScoringMode.Classic, HitResult.Meh, HitResult.Great, 156)]
+        [TestCase(ScoringMode.Classic, HitResult.Ok, HitResult.Great, 312)]
+        [TestCase(ScoringMode.Classic, HitResult.Good, HitResult.Perfect, 535)]
+        [TestCase(ScoringMode.Classic, HitResult.Great, HitResult.Great, 936)]
+        [TestCase(ScoringMode.Classic, HitResult.Perfect, HitResult.Perfect, 936)]
+        [TestCase(ScoringMode.Classic, HitResult.SmallTickMiss, HitResult.SmallTickHit, 0)]
+        [TestCase(ScoringMode.Classic, HitResult.SmallTickHit, HitResult.SmallTickHit, 225)]
+        [TestCase(ScoringMode.Classic, HitResult.LargeTickMiss, HitResult.LargeTickHit, 0)]
+        [TestCase(ScoringMode.Classic, HitResult.LargeTickHit, HitResult.LargeTickHit, 936)]
+        [TestCase(ScoringMode.Classic, HitResult.SmallBonus, HitResult.SmallBonus, 30)]
+        [TestCase(ScoringMode.Classic, HitResult.LargeBonus, HitResult.LargeBonus, 150)]
+        public void TestFourVariousResultsOneMiss(ScoringMode scoringMode, HitResult hitResult, HitResult maxResult, int expectedScore)
+        {
+            var minResult = new TestJudgement(hitResult).MinResult;
+
+            IBeatmap fourObjectBeatmap = new TestBeatmap(new RulesetInfo())
+            {
+                HitObjects = new List<HitObject>(Enumerable.Repeat(new TestHitObject(maxResult), 4))
+            };
+            scoreProcessor.Mode.Value = scoringMode;
+            scoreProcessor.ApplyBeatmap(fourObjectBeatmap);
+
+            for (int i = 0; i < 4; i++)
+            {
+                var judgementResult = new JudgementResult(fourObjectBeatmap.HitObjects[i], new Judgement())
+                {
+                    Type = i == 2 ? minResult : hitResult
+                };
+                scoreProcessor.ApplyResult(judgementResult);
+            }
+
+            Assert.IsTrue(Precision.AlmostEquals(expectedScore, scoreProcessor.TotalScore.Value, 0.5));
+        }
+
+        [TestCase(ScoringMode.Standardised, HitResult.SmallTickHit, 978_571)]
+        [TestCase(ScoringMode.Standardised, HitResult.SmallTickMiss, 914_286)]
+        [TestCase(ScoringMode.Classic, HitResult.SmallTickHit, 279)]
+        [TestCase(ScoringMode.Classic, HitResult.SmallTickMiss, 214)]
+        public void TestSmallTicksAccuracy(ScoringMode scoringMode, HitResult hitResult, int expectedScore)
```

Comment:
How does this test work? Should I be reading the test? Is this test still required? I thought the reason for this test had to do with classic scoring being broken for `maxAchievableCombo = 0`.

Reply:
This test isn't required anymore but it still works and tests behaviour that's special to small ticks (that they only affect accuracy portion)

Probably needs documentation similar to previous test